doc/source/programs/gdal_contour.rst

```diff
@@ -94,6 +94,7 @@ be on the right, i.e. a line string goes clockwise around a top.
 .. option:: -i <interval>
 
     Elevation interval between contours. Ignored if -fl is used.
+    Must specify either -i or -fl or -e.
```

Comment:
shouldn't that mention be moved in the above "description" part to avoid repeated it 3 times?

Reply:
One could say I am putting a warning label on each individual cigarette, instead of just the pack.

But that's the only way one can guarantee users won't come howling when they hit one of the booby trapped options...